packages/next-intl/src/plugin/types.tsx

```diff
@@ -1,6 +1,6 @@
 export type PluginConfig = {
   requestConfig?: string;
   experimental?: {
-    createMessagesDeclaration?: string;
+    createMessagesDeclarations?: Array<string>;
```

Comment:
Can we keep the existing signature to make the common case easy?

```suggestion
    /** A path to the messages file that you'd like to create a declaration for. In case you want to consider multiple files, you can pass an array of paths. */
    createMessagesDeclaration?: string | Array<string>;
```